### PR TITLE
Fix storagesysyem physical calc

### PR DESCRIPTION
**Pull request by bvered**

### CODE REVIEW

**Comment by bvered** [comment]
[Console - storage overview - fix physical capacity calcs](https://trello.com/c/57pkz1lF/33-console-storage-overview-fix-physical-capacity-calcs)

**Comment by bvered** [comment]
[Console - system overview - fix utilization - used capacity for system calc](https://trello.com/c/CUARxys5/38-console-system-overview-fix-utilization-used-capacity-for-system-calc)

File: src/constants/queries.ts [hunk 1]
```diff
@@ -169,21 +185,44 @@ export const BreakdownQueryMapODF = (label: string, queryType: string) => {
...
+      //     desc: "Used",
+      //   },
+      //   {
+      //     query: FlASHSYSTEM_QUERIES(
+      //       label,
+      //       StorageDashboardQuery.TotalFreeCapacity
+      //     ),
+      //     desc: "Available",
+      //   },
+      // ];
```
Comment (tyichye19) [inline src/constants/queries.ts:203]: I think we can remove these commented lines, they are not relevant anymore.

Comment (bvered) [reply src/constants/queries.ts]: Done

**Review by tyichye19 — APPROVED** [review]
LGTM

**Pull request merged**